ceph.spec.in

```diff
@@ -1173,11 +1173,6 @@ ${CMAKE} .. \
 %if 0%{with ocf}
     -DWITH_OCF=ON \
 %endif
-%ifarch aarch64 armv7hl mips mipsel ppc ppc64 ppc64le %{ix86} x86_64 s390x
```

Comment:
why? this matches with https://www.boost.org/doc/libs/1_72_0/libs/context/doc/html/context/architectures.html .

Reply:
Because we don't have a WITH_BOOST_CONTEXT option any more. Though I'll break it out into a separate commit.

Reply:
oic. thanks!